zetaclient/evm_client.go

```diff
@@ -26,8 +26,9 @@ import (
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
 
 	erc20custody "github.com/zeta-chain/protocol-contracts/pkg/contracts/evm/erc20custody.sol"
+	zetaconnector "github.com/zeta-chain/protocol-contracts/pkg/contracts/evm/zetaconnector.non-eth.sol"
```

Comment:
```suggestion
```
I think there is a build error because duplicated import

Reply:
I have removed duplicate imports.

Reply:
@lumtis @kingpinXD please check